content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx

```diff
@@ -22,7 +22,7 @@ The following example models a one-to-one self-relation:
 model User {
   id          Int     @id @default(autoincrement())
   name        String?
-  successorId Int?
+  successorId Int?    @unique
```

Comment:
I think lines 54 to 60 describe what is needed for a one-to-one self relation. And now it seems we also require the `@unique` on the scalar relation field, so maybe we should add that to the list?

Maybe an addition to the third list item:
> - One relation field must be backed by a foreign key. The `successor` field is backed by the `successorId` foreign key, which references a value in the `id` field. The `successorId` scalar relation field needs to be `@unique` to guarantee a 1:1 relation.

Reply:
I like the suggestion.

Reply:
Thanks, done